Makefile

```diff
@@ -42,15 +42,15 @@ check-for-cuda :
 
 .PHONY : lint
 lint :
-	flake8 ./scripts ./tests $(SRC)
```

Comment:
Is this intentional? Don't we want these checks on tests?  Oh, I think I get it - this change just makes it find all python files under the current directory?  I didn't realize it would work with a mixed (with non-python) directory like this.

Reply:
Yeup, this should find every python file in the repo

Reply:
Don't you wanna check only the changed files?